docker-compose.yml

```diff
@@ -93,6 +93,20 @@ services:
         aliases:
           - devm
 
+  image-manager:
```

Comment:
In retrospective, perhaps calling this service image-manager (mea culpa) is not a good idea (might be confused for actual images, like jpg files).

Reply:
Yeah, I also had that impression. Perhaps artifact-manager? or even firmware-manager...

Reply:
firmware-manager seems a good one.